.gitignore

```diff
@@ -8,7 +8,8 @@ bin/
 *.dylib
 
 config.toml
-
+.shared.env
```

Comment:
I can't use `.env-*` if I put samples as you mention below: `.env-issuer-example` @cneuro

Reply:
My proposal for env file naming would be:

- `.env-issue`r for the issuer API
- `.env-api` for the UI API only
- `.env-shared` for shared variables between UI and UI API (no overlap with .envapi)
- `.env-ui` for UI app only
  
To provide examples, it would be `.env-{DOMAIN}.example`, so `.env-issuer.example`, `.env-api.example` etc. That way they are consistent, sorted properly, easily distinguishable, atomized in their scope and can be gitignored straight-forwardly.